### ISSUE TITLE
HTTP links turn to HTTPS Links on doc.traefik.io get started page

### ISSUE THREAD

**Issue by mitnk** [traefik/traefik#8346]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you do?

read and follow quick start page: https://doc.traefik.io/traefik/getting-started/quick-start/

### What did you see instead?

The hype-link on `http://localhost:8080/api/rawdata` is HTTPS which make clicking on it 404.

### What version of Traefik are you using?

2.4

### What is your environment & configuration?

as on quick start page.

**Comment by mitnk** [comment]
![image](https://user-images.githubusercontent.com/177049/128685304-416da890-ff02-4f24-a171-913805d7e647.png)

See link hint on left-bottom of this image.

**Comment by mitnk** [comment]
The source doc file seems correct: https://github.com/traefik/traefik/edit/v2.4/docs/content/getting-started/quick-start.md:

```
You can open a browser and go to [http://localhost:8080/api/rawdata](http://localhost:8080/api/rawdata) to see Traefik's API rawdata ...
```

**Comment by santosh** [comment]
@mitnk Are you using any browser extension which is forcing http -> https?

Because with `curl -XGET --head http://localhost:8080/api/rawdata`, I see no 3XX response.

**Comment by mitnk** [comment]
It's not local browser, it's the doc host who rendering that link HTTPS.

**Comment by mpl** [comment]
yeah, mkdocs is automatically forcing links with https. But we found a workaround, we'll apply it soon.

**Comment by traefiker** [comment]
Closed by #8395.
